ProjectBank/Server.Tests/Controllers/ProjectsControllerTests.cs

```diff
@@ -0,0 +1,133 @@
+namespace Server.Tests;
+
+public class ProjectsControllerTests
+{
+
+    [Fact]
+    public async Task Post_creates_Project()
+    {
+        // Arrange
+        var toCreate = new CreateProjectDTO();
+        var created = new ProjectDTO(1, "Very nice project", "Active", "1", "A created project", DateTime.UtcNow, DateTime.UtcNow, new List<string>(), new List<UserDTO>());
+        var repository = new Mock<IProjectRepository>();

```

Comment:
We create "var repository = new Mock<IProjectRepository>();" in every test. We can not reuse the same one or? Maybe just clear it or whatever is needed

Reply:
Good point! Fixed this